modules/edu.gemini.seqexec.engine/src/main/scala/edu/gemini/seqexec/engine/Sequence.scala

```diff
@@ -46,6 +46,8 @@ object Sequence {
     done: List[Step]
   ) {
 
+    private val (toSkip, remaining): (List[Step], List[Step]) = pending.span(_.skipMark)
```

Comment:
`pendings` is separated into the leading steps to be skipped, and the rest.

Reply:
Wouldn't `span` stop at the first step to skip. What happens if you have multiple skipped steps?

Reply:
It stops at the first element that does not fulfill the condition. All the elements in the leading `List` are assured to pass the test, while the first element of the tailing list is assured to not pass the test.